core/src/replay_stage.rs

```diff
@@ -3087,6 +3082,7 @@ impl ReplayStage {
         replay_result_vec: &[ReplaySlotFromBlockstore],
         purge_repair_slot_counter: &mut PurgeRepairSlotCounter,
         my_pubkey: &Pubkey,
+        mut tbft_structs: Option<&mut TowerBFTStructures>,
```

Comment:
I doubt we'll have a third competing consensus algorithm, but what do you think about the following:
```rust
enum ConsensusAlgorith {
    TowerBft(TowerBFTStructures),
    // Same structure for now, but will be updated later
    Alpenglow(TowerBFTStructures),
}
```
Namely, I like the enum naming instead of `Some(_) = tower` and `None = AG`